language/stdlib/modules/LibraAccount.move

```diff
@@ -157,6 +158,14 @@ module LibraAccount {
         );
     }
 
+    fun should_track_limits(scrutinee: address, other_account: address): bool {
```

Comment:
I think it might be clearer for if we avoid introducing the concept of "scrutinee", which is context-dependent and thus potentially confusing for callers. If we just use `payer` and `payee` here, that will make the body of this function a bit bigger, but it will be easier for callers and will map very directly to the clear description you lay out in the PR summary:

```
VASP->any we check and update the sending account's account limits
any->VASP we check and update the receiving account's account limits
```